### PR TITLE
Improve benchmarking scripts

### PR DESCRIPTION
**Pull request by d-alonso**
To output CSV and to run multiple compilers

### CODE REVIEW

**Review by inaki-amatria — APPROVED** [review]
LGTM :-)

File: Benchmark/README.md [hunk 1]
```diff
@@ -14,9 +14,53 @@ to run all the available benchmarks, or run-<CHECK ID> to run a specific one.
 
 Or simply run the following command to do it automatically:
 ```
-./run-benchmarks.sh
+./run-benchmarks.py
```
Comment (alvrogd) [inline Benchmark/README.md:17]: We should list Python 3 as an optional dependency at the start of the `README`.

**Review by alvrogd — APPROVED** [review]
:fire:

File: Benchmark/README.md (hunk 1 above)
Comment (d-alonso) [reply Benchmark/README.md:18]: Yep, done

**Pull request merged**